broker/submissions/export_to_spreadsheet_service.py

```diff
@@ -6,11 +6,12 @@
 class ExportToSpreadsheetService:
 
     def __init__(self, ingest_api: IngestApi = None):
-        self.ingestapi = IngestApi() if not ingest_api else ingest_api
-        self.data_collector = DataCollector(self.ingestapi)
+        self.ingest_api = IngestApi() if not ingest_api else ingest_api
+        self.data_collector = DataCollector(self.ingest_api)
         self.downloader = XlsDownloader()
 
     def export(self, submission_uuid: str):
+        self.ingest_api.unset_token()
```

Comment:
Why are we removing the token?
We just have to get a valid token if it expired or close to expiration

Reply:
Ok, we discussed offline that this is a temp solution to able to go ahead with #13 dev ticket, so I am going to approve this